swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
+
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__(self, inventory=None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
+        
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        try:
+            self.inventory.remove(item)
+            return item
+        except Exception:
+            return False
+
+    def get_by_category(self, category):
+        items_by_category = []
+
+        for item in self.inventory:
+            if item.category == category:
+                items_by_category.append(item)
```

Comment:
Notice this pattern of 
```python
result_list = []
for element in source_list:
    if some_condition(element):
        result_list.append(element)
```

can be rewritten as a list comprehension as
```python
result_list = [element for element in source_list if some_condition(element)]
```

Which here would look like
```python
        items_by_category = [item for item in self.inventory if item.category == category]
```

At first, this may seem more difficult to read, but comprehensions are very common python style, so try getting used to working with them!